src/Tizen.Applications.NotificationEx/Tizen.Applications.NotificationEx/Manager.cs

```diff
@@ -0,0 +1,322 @@
+            /// <remarks>
+            /// Every request method such as Update, returns request id as a request result.
+            /// You can identify which request is in trouble with request id.
+            /// </remarks>
+            /// <since_tizen> 7 </since_tizen>
+            protected virtual void OnError(ErrorCode err, int requestId)
+            {
+            }
+
+            /// <summary>
+            /// Gets a notification item list.
+            /// </summary>
+            /// <returns>Notification item list. </returns>
+            /// <exception cref="InvalidOperationException">Thrown when fail to get notification list.</exception>
+            /// <exception cref="OutOfMemoryException">Thrown when fail to create a list because of the out of memory.</exception>
+            /// <since_tizen> 7 </since_tizen>
+            public IList<AbstractItem> Get()
+            {
+                IntPtr ptr;
+                int count;
+
+                ErrorCode err = Interop.NotificationEx.ManagerGet(_handle, out ptr, out count);
+                if (err != ErrorCode.None)
+                    ErrorFactory.ThrowException(err);
+                IList<AbstractItem> itemList = Util.IntPtrToList(ptr, count);
+                Interop.Libc.Free(ptr);
+
+                return itemList;
+            }
+
+            /// <summary>
+            /// Gets a notification item list by channel.
+            /// </summary>
+            /// <param name="channel"> A channel name. </param>
+            /// <returns>Notification item list. </returns>
+            /// <exception cref="InvalidOperationException">Thrown when fail to get notification list.</exception>
+            /// <exception cref="OutOfMemoryException">Thrown when fail to create a list because of the out of memory.</exception>
+            /// <since_tizen> 7 </since_tizen>
+            public IList<AbstractItem> Get(string channel)
+            {
+                IntPtr ptr;
+                int count;
+
+                ErrorCode err = Interop.NotificationEx.ManagerGetByChannel(_handle, channel, out ptr, out count);
+                if (err != ErrorCode.None)
+                    ErrorFactory.ThrowException(err);
+                IList<AbstractItem> itemList = Util.IntPtrToList(ptr, count);
+                Interop.Libc.Free(ptr);
+
+                return itemList;
+            }
+
+            /// <summary>
+            /// Updates a notification item.
+            /// </summary>
+            /// <param name="item"> A notification item to be updated. </param>
+            /// <returns>ID of the request. </returns>
+            /// <exception cref="InvalidOperationException">Thrown when fail to update notification.</exception>
+            /// <exception cref="ArgumentException">Thrown when the item parameter is null. </exception>
+            /// <since_tizen> 7 </since_tizen>
+            public int Update(AbstractItem item)
```

Comment:
How to use return value?

Reply:
If `Update` method is async operation, 
```suggestion
            public Task Update(AbstractItem item)
```